src/features/DisplayControls/Treatments/utils/TreatmentsUtils.js

```diff
@@ -121,6 +127,15 @@ export const isIPDDrugOrder = (drugOrderObject) => {
   return drugOrderObject.drugOrder.careSetting === "INPATIENT";
 };
 
+export const isDrugOrderStoppedWithoutAdministration = (drugOrderObject) => {
+  return (
+    drugOrderObject.drugOrder.dateStopped !== null &&
+    (drugOrderObject.drugOrderSchedule === null ||
+      drugOrderObject.drugOrderSchedule?.medicationAdministrationStarted ===
+        false)
+  );
```

Comment:
Here We can do:
return drugOrderObject.drugOrder.dateStopped && (!drugOrderObject.drugOrderSchedule || !drugOrderObject.drugOrderSchedule?.medicationAdministrationStarted)